doc/nrf/known_issues.rst

```diff
@@ -912,6 +912,11 @@ NCSDK-6832: SMP Server sample fails upon initialization
 
   **Workaround:** Set :option:`CONFIG_MAIN_STACK_SIZE` to ``2048``.
 
+.. rst-class:: v1-4-99-dev1
```

Comment:
Can you move it to the top of the section (since it's new)?

Reply:
moved to the top